src/deezer/client.py

```diff
@@ -605,3 +605,11 @@ def search_artists(
             strict=strict,
             ordering=ordering,
         )
+
+    """"
+    Search track matching the given isrc.
+    :param isrc: the isrc to search for.
+    """
+
+    def search_isrc(self, isrc: str = "") -> Track:
+        return self.request("GET", f"/track/isrc:{isrc}", resource_type=Track)
```

Comment:
I don't see any mention of this endpoint in the [Deezer API docs](https://developers.deezer.com/api/track#actions). Where did you get that from? Is that even working?

Reply:
I just fuzzed the api, yes, it's working. You can check here https://api.deezer.com/track/isrc:ITUM72000710

Reply:
If that's not documented, I don't aim to maintain a method for it, sorry. There is a escape hatch for users who want to use it, and that's probably good enough.